script.xbmc.boblight/default.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*- 
+'''
+    Boblight for Kodi
+    Copyright (C) 2012 Team XBMC
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+'''
+import xbmc
+import xbmcaddon
+import xbmcgui
+import os
+
+__addon__      = xbmcaddon.Addon()
+__cwd__        = __addon__.getAddonInfo('path')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__    = __addon__.getAddonInfo('version')
+__icon__       = __addon__.getAddonInfo('icon')
+__ID__         = __addon__.getAddonInfo('id')
+__language__   = __addon__.getLocalizedString
+
+__profile__    = xbmc.translatePath( __addon__.getAddonInfo('profile') )
+__resource__   = xbmc.translatePath( os.path.join( __cwd__, 'resources', 'lib' ) )
+
+sys.path.append (__resource__)
```

Comment:
Please avoid using sys.path.append and use relative module imports

`from resources.xx.xx import xx`